consul/base.py

```diff
@@ -48,7 +48,7 @@ class Check:
     There are three different kinds of checks: script, http and ttl
     """
     @classmethod
-    def script(klass, args, interval):
+    def script(klass, args, interval, deregister=None):
```

Comment:
Shouldn't we keep the exact same semantic, ie. name the argument `deregister_critical_service_after`?

Reply:
Hmm, OK you can forget this request, since it's not this way for other checks (http, tcp).